assets/js.js

```diff
@@ -0,0 +1,210 @@
+const errorMessage = document.querySelector('.error-message');
+const searchBtn = document.querySelector('.search-btn')
+
+
+//BEGIN - GET ALL GAME
+const getAllGames = async () => {
+    try {
+        const url = `https://cs-steam-game-api.herokuapp.com/games
+        `
+        //here is how we add a dynamic value (API KEY) to the url
+        const res = await fetch(url)
+        const data = await res.json()
+        // console.log("data", data) //have a look the retrieved data
+        return data
+    } catch (error) {
+        console.log('error', error)
+    }
+
+}
+
+const renderGames = async () => {
+    document.querySelector('#nav-search').value =''
+    try {
+        errorMessage.style.display = 'flex'
+        const data = await getAllGames()
+        const gamesList = document.getElementById("content")
+        const ulGamesList = gamesList.children[1]
```

Comment:
I'm not a big fan of `children`, you can give the `div` you wanna take an `id`, then use `document.getElementById` to be at the place you want